### PR TITLE
Fix confusing note

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#1224

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/classifier-getting-started-with.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Sensitivity and retention labels can then be automatically applied to make the c
...
 
 > [!IMPORTANT]
-> Both types of classifiers are available as a condition for [auto-apply retention label policy based on a condition](labels.md#applying-a-retention-label-automatically-based-on-conditions) and [Communication compliance](communication-compliance.md).
+> Both **Automated pattern matching** and **Trainable classifiers** are available as a condition for [auto-apply retention label policy based on a condition](labels.md#applying-a-retention-label-automatically-based-on-conditions) and [Communication compliance](communication-compliance.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/classifier-getting-started-with.md]:
```suggestion
> Both **Automated pattern matching** and **Trainable classifiers** are available as conditions for [applying a retention label automatically based on conditions](labels.md#applying-a-retention-label-automatically-based-on-conditions) and [communication compliance](communication-compliance.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by mypil** [comment]
@MaratMussabekov  - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@chrfox - PR has been copy edited and is ready for final review. This is based on issue #1224.


Thank you.

cc: @kenwith

**Comment by MaratMussabekov** [comment]
replaced by  #1310

**Pull request closed without merging**